File: model/src/main/java/net/artux/pda/model/items/ArtifactModel.java
package net.artux.pda.model.items;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class ArtifactModel extends WearableModel {

    private int anomalyId;
    private int health;
    private int radio;
    private int damage;
    private int bleeding;
    private int thermal;
    private int chemical;
    private int endurance;
    private int electric;

}

File: model/src/main/java/net/artux/pda/model/items/ArtifactModel.kt
package net.artux.pda.model.items

data class ArtifactModel(
    var anomalyId: Int = 0,
    var health: Int = 0,
    var radio: Int = 0,
    var damage: Int = 0,
    var bleeding: Int = 0,
    var thermal: Int = 0,
    var chemical: Int = 0,
    var endurance: Int = 0,
    var electric: Int = 0,
) : WearableModel() {

}